modules/core/src/test/java/org/apache/ignite/hlc/HybridClockTest.java

```diff
@@ -48,7 +48,7 @@ public void afterEach() {
     public void testNow() {
         clockMock = mockToEpochMilli(100);
 
-        HybridClock clock = new HybridClock();
+        HybridClock clock = new HybridClock(new SystemTimeProvider());
```

Comment:
Passing a specific implementation looks really weird to me. Do you expect other implementations to extend SystemTimeProvider? This doesn't look correct. Why not to implement getter as a closure and pass it like new HybridClock(SystemTimeProvider::getPhysicalTime)

Reply:
I removed a time provider. I created it for tests but it is not needed now.